daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_dev/daml_lf_1.proto

```diff
@@ -57,23 +57,23 @@ message PackageRef {
     Unit self = 1;
 
     // A `Package identifier` for an imported Package.
-    // A ascii7 lowercase hex-encoded package identifier. This refers
-    // to the DAML LF Archive Hash.
-    string package_id = 2;
+    // *Must be a list of valid packageId string*
+    string package_id_str = 2; // *Available in versions < 1.dev*
 
     // An index into `interned_package_ids` of the Package containing
     // this reference.
-    int32 interned_id = 3; // *Available in versions >= 1.6*
+    // *Must be a valid interned package Id string*
+    int32 package_id_interned_str = 3; // *Available in versions >= 1.6*
   }
 }
 
 // A `name`, e.g. Util.Either.isLeft
+// *Available in version < 1.dev*
 message DottedName {
-  // *Must be a non-empty list of a valid identifiers unless `segments_interned_id` is set.*
+
+  // *Must be a non-empty list of a valid identifiers*
   repeated string segments = 1;
 
-  // *Must be set if and only if `segments` is an empty list.*
-  int32 segments_interned_id = 2; // *Available in versions >= 1.dev*
```

Comment:
add `reserved 2`?

Reply:
No needs since, it was never frozen in stable LF version.